doc/en/tmpdir.rst

```diff
@@ -192,6 +192,7 @@ You can override the default temporary directory setting like this:
 
     pytest --basetemp=mydir
 
+**Warning**: The contents of ``mydir`` will be deleted.
```

Comment:
```suggestion
.. warning:: The contents of ``mydir`` will be deleted.
```

To have a proper warning box, see the [Sphinx cheatsheet](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#colored-boxes-note-seealso-todo-and-warnings).

Reply:
Oh, I see. Thanks, modified the comment.